product/roundhouse/infrastructure.app/ConfigurationPropertyHolder.cs

```diff
@@ -37,7 +37,7 @@ public interface ConfigurationPropertyHolder
         string VersionTableName { get; set; }
         string ScriptsRunTableName { get; set; }
         string ScriptsRunErrorsTableName { get; set; }
-        string EnvironmentName { get; set; }
+        string EnvironmentNames { get; set; }

```

Comment:
Same goes here. This will break existing API users. You must continue to provide EnvironmentName (and possibly give a deprecation warning.